test/integration/api/odata.js

```diff
@@ -242,7 +258,7 @@ describe('api: /forms/:id.svc', () => {
           .then(({ body }) => {
             body.should.eql({
               '@odata.context': 'http://localhost:8989/v1/projects/1/forms/doubleRepeat.svc/$metadata#Submissions.children.child',
-              '@odata.nextLink': "http://localhost:8989/v1/projects/1/forms/doubleRepeat.svc/Submissions('double')/children/child?%24skip=2",
+              '@odata.nextLink': "http://localhost:8989/v1/projects/1/forms/doubleRepeat.svc/Submissions(%27double%27)/children/child?%24skip=2",
```

Comment:
I encode the instance ID above, but I don't encode `'`, so it's very interesting to me that `'` becomes `%27` here. I think what's happening is that `%27` was part of the `originalUrl`, because supertest is encoding the `'` (though I don't think it needs to).

Reply:
yeah this is probably what led me to do the unencoding at all.